.github/workflows/test_on_push.yml

```diff
@@ -72,61 +114,107 @@ jobs:
         # sometimes gfortran cannot be found, so reinstall gcc just to be sure
         run: |
           brew analytics off
-          brew install graphviz libomp
+          brew install graphviz libomp openblas
```

Comment:
```suggestion
          brew install graphviz libomp
```

Same question as above

Reply:
We probably don't need it. I will test it and let you know

Reply:
Yes we don't need it. I already removed it from the test workflows and they pass